public/js/components/Rep.js

```diff
@@ -5,8 +5,8 @@ const Grip = require("../lib/devtools/client/shared/components/reps/grip").Grip;
 
 require("../lib/devtools/client/shared/components/reps/reps.css");
 
-function renderRep({ object }) {
-  return Rep({ object, defaultRep: Grip });
+function renderRep({ object, mode }) {
```

Comment:
Looks like it uses the default output? https://dxr.mozilla.org/mozilla-central/source/devtools/client/shared/components/test/mochitest/test_reps_object.html#79